src/components/ItemTableRow.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+type PropType = {
+  item: any;
+  index: number;
+};
+
+function ItemTableRow({ item, index }: PropType) {
+  return (
+    <tr>
+      <td>{index + 1}</td>
+      <td>{item.name}</td>
+      <td>${item.price}</td>
```

Comment:
$ not needed

Reply:
That is for dollar sign on price

Reply:
lol. kinda ambigious to code. need to properly separated. for now it's okay, I will be merging this PR.